guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/repositories/GitMetadataStore.java

```diff
@@ -37,6 +37,14 @@ void write( String name,
                 String origin );
 
     /**
+     * Writes the meta information about a repository.
+     * @param name The name of the repository in organizationalUnit/repositoryName format
+     * @param metadata The metadata object that stores information about repository
+     */
+    void write( String name,
```

Comment:
Personally I'm not a fan of the parameter ```name``` which needs to be in some magical "...organizationalUnit/repositoryName format..." but I see some other interface methods take this magical value too. Personally I'd prefer to have supplied the OU Identifier and Repository Identifier and let the magic concatenation occur inside the implementation..